internal/cli/atlas/deployments/list.go

```diff
@@ -196,12 +190,16 @@ func ListBuilder() *cobra.Command {
 
 			opts.defaultSetter.OutWriter = cmd.OutOrStdout()
 
-			opts.podmanClient = podman.NewClient(log.IsDebugLevel(), log.Writer())
-			return opts.podmanClient.Ready(cmd.Context())
+			opts.PodmanClient = podman.NewClient(log.IsDebugLevel(), log.Writer())
+			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run(cmd.Context())
 		},
+		PostRunE: func(cmd *cobra.Command, args []string) error {
+			opts.PostRunMessages()
```

Comment:
nit: we have this pattern for post run, commands with --watch normally have it
```suggestion
			return opts.PostRun(cmd.Context())
```

Reply:
thank you! will add